frontend/src/routes/Home/Search/searchDefinitions.tsx

```diff
@@ -1511,7 +1515,6 @@ export function CreateExternalLink(item: any) {
     if (item.consoleURL) {
         return (
             <a target="_blank" rel="noopener noreferrer" href={`${item.consoleURL}`}>
-                {/* TODO Not translating - caused issue: https://github.com/open-cluster-management/backlog/issues/9184 */}
```

Comment:
Have you investigated https://github.com/open-cluster-management/backlog/issues/9184 and whether translating this string would cause an issue?

Reply:
Also, you only removed the comment but did not add translation...